starkyModules/metadataRoles.ts

```diff
@@ -0,0 +1,83 @@
+import { retrieveAssets } from "../aspect/retrieveAssets";
+
+import { StarkyModuleConfig, StarkyModuleField } from "./types";
+
+export const name = "Metedata Roles";
```

Comment:
same typo 😉

Reply:
actually can we rename it `ERC-721 Metadata` ? Don't need the roles here, everything in Starky is about giving roles :)
We could probably change the name of the file also.